python/doc/pyplots/TensorProductExperiment.py

```diff
@@ -0,0 +1,24 @@
+import openturns as ot
+from matplotlib import pyplot as plt
+from openturns.viewer import View
+ot.RandomGenerator.SetSeed(0)
+
+collection = ot.WeightedExperimentCollection(2)
+collection[0] = ot.GaussProductExperiment(ot.Uniform(0.0, 1.0), [3])
+collection[1] = ot.GaussProductExperiment(ot.Uniform(0.0, 1.0), [5])
+experiment = ot.TensorProductExperiment(collection)
```

Comment:
doesnt it work using a regular python list/tuple here instead of WeightedExperimentCollection ?

Reply:
Yes, it works by the magic of the Python binding. I update the test to remove that, and use a plain list.